v2/devices/surya.yml

```diff
@@ -0,0 +1,132 @@
+name: "Xiaomi Redmi Note 9 Pro/Pro Max/9S - Poco M2 Pro"
+codename: "surya"
+aliases: []
+formfactor: "phone"
+doppelgangers: []
+user_actions:
+  recovery:
+    title: "Reboot to Recovery"
+    description: "With the device powered off, hold Volume Up + Power."
+    image: "phone_power_up"
+    button: true
+  bootloader:
+    title: "Reboot to Bootloader"
+    description: "With the device powered off, hold Volume Down + Power."
+    image: "phone_power_down"
+    button: true
+  confirm_model:
```

Comment:
You should be able to drop this, unless the same device has multiple models and not all of them are supported.

Reply:
I assume you only mean `confirm_model:`?

Reply:
Yes, the `confirm_model` section.

There was also a discussion regarding the unlocking of devices.
https://github.com/ubports/installer-configs/commit/54f34fe1a2952c527cdb9407be50a5fdd78d9ed1

Basically you can drop the bootloader unlock user_action as well and if you need a code, you can set `code_url` like here: https://github.com/ubports/installer-configs/blob/54f34fe1a2952c527cdb9407be50a5fdd78d9ed1/v2/devices/FP3.yml#L31